src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -109,6 +109,14 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('Whether to set the Content-Language HTTP header on the Response using the Request locale.')
                     ->defaultFalse()
                 ->end()
+                ->booleanNode('handle_partial_denormalization_exception')
+                    ->info('Whether to convert Exceptions that occurs during Denormalization to ValidationFailedException.')
+                    ->defaultFalse()
+                ->end()
+                ->booleanNode('handle_validation_failed_exception')
+                    ->info('Whether to convert Validation Exceptions to Http Bad Request 400 Response.')
+                    ->defaultFalse()
+                ->end()
```

Comment:
isnt this the expected behavior when resolving controller arguments? other resolvers always throw HttpException

Reply:
I would say, it should be true by default.

Reply:
`true` by default can be some kind of BC. We can deprecate not setting this value and say they will be `true` by default since 7.0.

Reply:
i believe argument resolvers should always throw HttpException